iot/README.md

```diff
@@ -1,80 +1,7 @@
-<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
-Cloud Platform logo" title="Google Cloud Platform" align="right" height="96"
-width="96"/>
+# Deprecation Notice
 
-# Google Cloud IoT Core API Samples
+* <h3>Google Cloud IoT Core will be <a href="https://cloud.google.com/iot/docs/release-notes#August_16_2022">retired as of August 16, 2023</a>.</h3>
```

Comment:
Please use markdown syntax rather than html tags.

Reply:
We follow the same template in all docs samples repos. I admit that it does not strictly follow the standard. If it is not  a fundamental requirement for you, I appreciate it if we can leave it "as is" until we clean up the repo (probably in 2024) completely.

Reply:
It's ugly and more difficult for customers to read in source. I don't understand why you'd want it this way, rather than the exact same content but in clean markdown that is much easier to read. But I suppose we don't have a hard requirement or anything.